pkg/backend/backend.go

```diff
@@ -155,7 +155,7 @@ type Backend interface {
 	ValidateStackName(s string) error
 
 	// DoesProjectExist returns true if a project with the given name exists in this backend, or false otherwise.
-	DoesProjectExist(ctx context.Context, projectName string) (bool, error)
+	DoesProjectExist(ctx context.Context, projectName string, orgName string) (bool, error)
```

Comment:
I would flip the argument order here. Other APIs do org, project, stack (decreasing order of scope). It'll be easy to make a mistake here and flip the order.

Also, perhaps add a comment on whether orgName is required or optional.